### DISCUSSION TITLE
Scaling clarification

### DISCUSSION THREAD

**Discussion by maitrungduc1410** [taskforcesh/bullmq#1547] [General]
Hi all,

I created this discussion just want to clarify few points. In production usually we'll more than 1 instance of server, let say 10 instances of Nodejs server, and it can be dynamically changed with tools like Kubernetes. So:
1. if I have more than 1 instance, there'll be chance where job is added from instance A but then will be handled by worker of instance B?
2. in case of repeatable job (cron job), if I add job from instance A, so from that time, does the job will be processed by all instances (A and B)? If not, can you pls explain briefly how it works?

Thanks

**Comment by manast** [comment]
A job will be processed just once by any worker that is connected to a given queue. More workers are useful for redundancy and scaling up.
